### DISCUSSION TITLE
发现一款疑似修改小狼毫的输入法

### DISCUSSION THREAD

**Discussion by MokOopsing** [rime/home#705] [General]
该输入法名为极简输入法，官网http://geek-input.officeoncloud.cn/ ，这个输入法改的小狼毫，但是未看到有开放源码，应该涉及侵权

**Comment by lotem** [comment]
牛啊。界面很主流啊，比小狼毫厲害。
是國內的小作坊吧。
臺灣人早就仿製過。
也無奈何。
如果有用戶論壇什麼的，可以借勢去宣傳 RIME：
好用的輸入法懂我心意，請認準式恕堂註冊商標。

**Reply by ricoeur** [reply]
合肥云表信息科技有限公司
https://www.tianyancha.com/company/3024314518
https://bbs.kafan.cn/thread-2193304-1-1.html

**Comment by poorich** [comment]
某官方组织还十分看重它
见 http://www.catticenter.com/ggl/2367 条目3

**Comment by oishi81** [comment]
易用么？rime就是上手门槛比较高，易用性能做好的话就好了。  可以告它侵权！

**Reply by ghost** [reply]
在中国，所有开源协议形同虚设
